bip-p2qrh.mediawiki

```diff
@@ -0,0 +1,609 @@
+<pre>
+  BIP: TBD
+  Title: QuBit: SegWit version 3 spending rules (P2QRH)
+  Author: Hunter Beast <hunter@surmount.systems>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-TBD
+  Status: Draft
+  Type: Standards Track
+  License: BSD-3-Clause
+  Created: 2024-06-08
+</pre>
+
+__TOC__
```

Comment:
```suggestion
```
The TOC is generated automatically in the richtext view:

![image](https://github.com/user-attachments/assets/466e33a6-64e1-4b5d-b857-8681d4d3565e)